### DISCUSSION TITLE
Bug - rustdesk-1.2.0-x86_64-pc-windows-msvc.exe - CTRL+ALT+DEL to unlock guest not working in rustdesk-1.2.0-x86_64-pc-windows-msvc.exe

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#2575] [General]
If both the Host and Guest are running 1.2.0 
When you lock the guests screen, if you select the Lighten Bolt and then choose Insert CTRL+ALT+DEL. It will not unlock the guest.
If the Host is 1.9 and guest is 1.20 The function is fine.

**Comment by rustdesk** [comment]
@fufesou

**Comment by fufesou** [comment]
@gittygittymeow Thanks for your feedback.
https://github.com/rustdesk/rustdesk/pull/2576

**Reply by ghost** [reply]
Great work, just tried todays nightly and it looks to be fixed now. Thanks!
